pkg/policy/api/selector.go

```diff
@@ -39,6 +41,10 @@ type EndpointSelector struct {
 	//
 	// Kept as a pointer to allow EndpointSelector to be used as a map key.
 	requirements *k8sLbls.Requirements
+
+	// cachedString is the cached string representation of this EndpointSelector
+	// +k8s:deepcopy-gen=false
```

Comment:
This is the kicker - generating deepcopy functionality for an unsafe.Pointer breaks :/

Reply:
Likely it does not matter, as we are creating deep copies only when we intend to change the selector (e.g., when folding in requirements), and in those cases we would need to nil the pointer anyway